decentraland/unity-renderer
# feat: nft emotes loop property integrated to avatar animator

## What does this PR change?

This PR modifies avatar animation controller to be able to loop certain animation clips coming from emotes (including NFT emotes) and having loop property enabled.

## How to test the changes?

After NFT emotes flow completion PR will be merged we need to merge dev to this branch and add some custom NFT emotes with loop property enabled and then check if the animation actually loops in the game. Emotes with loop property disabled should not loop. It would be also good to check animation transitions from looping emote to other animations like jumping or running.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
